### DISCUSSION TITLE
Link on Oqtane.org site top of page to GitHub repo

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.framework#2723] [Ideas]
I would like a link at the top of the main content of the Oqtane.org site maybe next to Get Started.

Reason is I always type it in my browser it pops up first then I have to scroll down if I select it by hitting enter.  I really just want to get here.  

I have to dig for finding the repo.

On another note I have a PR to share that should fix an issue with alias sites permissions on deck.

**Comment by sbwalker** [comment]
Please use a standard browser bookmark if you wish for a convenient method to access the GitHub repo
